server/src/main/java/com/defold/extender/PlatformConfig.java

```diff
@@ -11,7 +11,7 @@
     public String exePrefix; // deprecated
     public String exeExt; // deprecated
     public String writeLibPattern;
-    public String writeShLibPattern = new String();
+    public String writeShLibPattern;
```

Comment:
Having it as "null" makes it easier to check for overrides between the platform config instances.